scripts/utils/constructProposalCalldata.ts

```diff
@@ -15,6 +16,10 @@ type ExtendedAlphaProposal = {
   description: string;
 };
 
+type PodConfig = {
```

Comment:
Use interfaces over types wherever possible, as they're capable of introspection and not just aliases

Reply:
Nice idea for objects for sure, will update